docs/core/config/auth/userdb.md

```diff
@@ -120,7 +120,7 @@ These fields can be returned the exact same way as `uid`, `gid`, and `home`
 fields.
 
 It's also possible to override settings from `dovecot.conf`. For example the
-[[setting,mail_path]] and [[setting,quota_rule]] settings are commonly
+[[setting,mail_path]] and [[setting,quota_storage_size]] settings are commonly
```

Comment:
There is another reference to `[[setting,quota_rule]]` that needs replacement, line 199.